### PR TITLE
PXP-8078 - PROD IBD deploy SOWER

### PR DESCRIPTION
**Pull request by xritter1**
Link to Jira ticket if there is one:

### Environments
IBD-PROD

### Description of changes
deploy SOWER

### CODE REVIEW

File: ibdgc.datacommons.io/manifest.json [hunk 1]
```diff
@@ -145,6 +146,291 @@
...
   },
+    "sower": [
+    {
+      "name": "ingest-metadata-manifest",
+      "action": "ingest-metadata-manifest",
+      "activeDeadlineSeconds": 86400,
+      "serviceAccountName": "jobs-qa-ibd-planx-pla-net",
+      "container": {
+        "name": "job-task",
+        "image": "quay.io/cdis/metadata-manifest-ingestion:integration202105",
```
Comment (atharvar28) [inline ibdgc.datacommons.io/manifest.json:157]: can you change the image from `integration-branch` to use a `2021.02`?

File: ibdgc.datacommons.io/manifest.json [hunk 2]
```diff
@@ -145,6 +146,291 @@
...
+      ],
+      "restart_policy": "Never"
+    },
+    {
+      "name": "get-dbgap-metadata",
+      "action": "get-dbgap-metadata",
+      "serviceAccountName": "jobs-qa-ibd-planx-pla-net",
+      "container": {
+        "name": "job-task",
+        "image": "quay.io/cdis/get-dbgap-metadata:integration202105",
```
Comment (atharvar28) [inline ibdgc.datacommons.io/manifest.json:197]: similarly here

File: ibdgc.datacommons.io/manifest.json [hunk 3]
```diff
@@ -145,6 +146,291 @@
...
+      ],
+      "restart_policy": "Never"
+    },
+    {
+      "name": "manifest-indexing",
+      "action": "index-object-manifest",
+      "serviceAccountName": "jobs-qa-ibd-planx-pla-net",
+      "container": {
+        "name": "job-task",
+        "image": "quay.io/cdis/manifest-indexing:integration202105",
```
Comment (atharvar28) [inline ibdgc.datacommons.io/manifest.json:227]: similarly here

File: ibdgc.datacommons.io/manifest.json [hunk 4]
```diff
@@ -145,6 +146,291 @@
...
+      ],
+      "restart_policy": "Never"
+    },
+    {
+      "name": "indexd-manifest",
+      "action": "download-indexd-manifest",
+      "serviceAccountName": "jobs-qa-ibd-planx-pla-net",
+      "container": {
+        "name": "job-task",
+        "image": "quay.io/cdis/download-indexd-manifest:integration202105",
```
Comment (atharvar28) [inline ibdgc.datacommons.io/manifest.json:267]: here too!

File: ibdgc.datacommons.io/manifest.json [hunk 5]
```diff
@@ -145,6 +146,291 @@
...
+        }
+      ],
+      "restart_policy": "Never"
+    },
+    {
+      "name": "pelican-export",
+      "action": "export",
+      "container": {
+        "name": "job-task",
+        "image": "quay.io/cdis/pelican-export:integration202105",
```
Comment (atharvar28) [inline ibdgc.datacommons.io/manifest.json:306]: here too!

File: ibdgc.datacommons.io/manifest.json [hunk 6]
```diff
@@ -145,6 +146,291 @@
...
+        }
+      ],
+      "restart_policy": "Never"
+    },
+    {
+      "name": "pelican-export-files",
+      "action": "export-files",
+      "container": {
+        "name": "job-task",
+        "image": "quay.io/cdis/pelican-export:integration202105",
```
Comment (atharvar28) [inline ibdgc.datacommons.io/manifest.json:370]: here!

File: ibdgc.datacommons.io/manifest.json (hunk 1 above)
Comment (xritter1) [inline ibdgc.datacommons.io/manifest.json]:
```suggestion
        "image": "quay.io/cdis/metadata-manifest-ingestion:2021.02",
```

**Review by atharvar28 — APPROVED** [review]
lgtm 👍

**Pull request merged**